.github/workflows/auto-merge-dockers.yml

```diff
@@ -0,0 +1,25 @@
+name: Docker Update Auto Merge
+on:
+  pull_request:
+    types: [opened, edited]
+  schedule:
+    - cron: '0 11 * * *'
+    - cron: '00 14 * * *'
```

Comment:
Not sure I understand the logic behind the timings here, why don't we run straight after the PRs were open?

Reply:
Cause we are running checks (lint, validate) it takes some time for it to finish.
So I'm running this twice, once regular, and the second time usually after I fix things

Reply:
At what time does the regular flow starts? 11 sounds very late to me

Reply:
It start at 8 in the morning, but as the moment we're still running code.pan.run so it takes time. 
I can change the timestamps if you think it will be better
I will definitely change them after we resolve the build for the flow